### DISCUSSION TITLE
The structure in the TeX panel show an unexpected zero.

### DISCUSSION THREAD

**Discussion by LordWolfenstein** [James-Yu/LaTeX-Workshop#3414] [Q&A]
## Preliminary questions [Required]
When you use \part{} in a article the TeX tab fills in a `.0` between the part and the section. 

Creating documents till works perfectly. This is just an eye soar.

### Disable all the other extensions except for LaTeX Workshop, restart VS Code, and check that you still see this issue. [Required]
No.

## Describe the bug [Required]
Articles do not have chapters so I think it is looking for a chapter that is not there.

### To Reproduce
Create this latex document and click on TeX on the left in VSCode.
```
\documentclass{article}
\title{sandbox}
\begin{document}
\maketitle
\tableofcontents
\part{first}
\section{aaa}
\section{bbb}
\part{second}
\section{ccc}
\section{ddd}
\end{document}
```

![image](https://user-images.githubusercontent.com/27807240/181946000-cf4a108f-c0ed-4e60-80b5-458528591cc2.png)

### Expected behavior

I do not expect it to have a `.0` between part and section number in the panel to the left in the screen shoot.

**Comment by James-Yu** [comment]
You can define your hierarchy in `latex-workshop.view.outline.sections`. Skipped levels are indexed by `0`.
